packages/headless/src/features/document-suggestion/document-suggestion-state.ts

```diff
@@ -0,0 +1,44 @@
+import {CaseAssistAPIErrorStatusResponse} from '../../api/service/case-assist/case-assist-api-client';
+import {DocumentSuggestion} from '../../api/service/case-assist/get-document-suggestions/get-document-suggestions-response';
+
+export const getDocumentSuggestionInitialState =
+  (): DocumentSuggestionState => ({
+    enabled: false,
```

Comment:
The `enabled` flag should not be there anymore since the enable/disable actions have been removed.